### PR TITLE
Fixed grammatical errors

### PR DESCRIPTION
**Pull request by ConnorChase29**
**Summary**

**Test plan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,19 +17,19 @@
 
 ---
 
-**Fast:** Yarn caches every package it downloads so it never needs to download the same package again. It also parallelizes operations to maximize resource utilization so install times are faster than ever.
+**Fast:** Yarn caches every downloaded package, so it never needs to download the same package again. It also structures operations in a parallel format to maximize resource utilization, meaning install times become even faster.
```
Comment (BYK) [inline README.md]: How about the following instead:

>Yarn caches every package it has downloaded, so it never needs to download the same package again. It also does almost everything concurrently to maximize resource utilization. This means even faster installs.

File: README.md [hunk 2]
```diff
@@ -17,19 +17,19 @@
...
 
-**Reliable:** Using a detailed, concise lockfile format and a deterministic algorithm for installs, Yarn is able to guarantee that an install that worked on one system will work exactly the same way on any other system.
+**Reliable:** Using a detailed, concise, lockfile format and a deterministic algorithm for installments, Yarn is able to guarantee that any install that works on one system will work exactly the same on another system.
```
Comment (BYK) [inline README.md]: Don't think `installments` conveys the right meaning here. How about `installations`?

File: README.md [hunk 3]
```diff
@@ -17,19 +17,19 @@
...
 ## Features
 
-* **Offline Mode.** If you've installed a package before, you can install it again without any internet connection.
-* **Deterministic.** The same dependencies will be installed in the same exact way on any machine, regardless of install order.
-* **Network Performance.** Yarn efficiently queues up requests and avoids request waterfalls in order to maximize network utilization.
-* **Network Resilience.** A single request failing won't cause an install to fail. Requests are retried upon failure.
-* **Flat Mode.** Yarn resolves mismatched versions of dependencies to a single version to avoid creating duplicates.
+* **Offline Mode.** If you've installed a package before, then you can install it again without an internet connection.
+* **Deterministic.** The same dependencies will be installed in the same exact way on any machine, regardless of installation order.
+* **Network Performance.** Yarn efficiently queues requests and simultaneously avoids request waterfalls in order to maximize network utilization.
```
Comment (BYK) [inline README.md]: I don't think we need `simultaneously` here.

File: README.md [hunk 4]
```diff
@@ -17,19 +17,19 @@
...
+* **Network Resilience.** A single request that fails will not cause the entire installation to fail. Requests are automatically re-tried upon failure.
```
Comment (BYK) [inline README.md]: I thought `retry` and `retried` were valid and Google says they are. I'd keep `retried` for this.

File: README.md [hunk 5]
```diff
@@ -17,19 +17,19 @@
...
+* **Flat Mode.** Yarn resolves mismatched versions of dependencies and uniformly switches to a single version to avoid duplicates.
```
Comment (BYK) [inline README.md]: Don't think the changes in this sentence reflect the original meaning and/or add any clarification.

**Comment by arcanis** [comment]
Small note: when you're done, please rebase with master so that the tests pass and we can merge it :)

**Comment by ConnorChase29** [comment]
Dear Burak,

Thank you for reviewing my changes, appreciate it! I will definitely revise
and push it through master branch.

Best,
Connor R. Chase

**Comment by BYK** [comment]
Closing in favor of #4211.

**Pull request closed without merging**